content/en/docs/concepts/policy/admission-controllers/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Admission Controllers
+content_type: concept
+weight: 20
+description: An admission controller runs in the API server and can validate or mutate API requests. For example, the `AlwaysPullImages` admission controller modifies a new Pod to set the image pull policy to `Always`. [More...](/docs/concepts/policy/admission-controllers/)
```

Comment:
Admission controllers aren't policies. _Some_ admission controllers are the implementation mechanism for policies.

Reply:
clarified - added that some admission controllers act as policies